### PR TITLE
Updating codeowners

### PR DESCRIPTION
**Pull request by RachalCassity**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

Updating CODEOWNERS

### CODE REVIEW

File: .github/CODEOWNERS
Comment (rmtolmach) [inline .github/CODEOWNERS:132]: I see `spec/requests/v1/higher_level_reviews` is owned by @department-of-veterans-affairs/benefits-decision-reviews-be, so it makes sense to me to have them own _all_ `higher_level_reviews` files like you're doing in this PR.

File: .github/CODEOWNERS
Comment (rmtolmach) [inline .github/CODEOWNERS:200]: Are all of these lines being deleted because `app/models/bgs_dependents` will take care of everything?

File: .github/CODEOWNERS
Comment (rmtolmach) [inline .github/CODEOWNERS:402]: How did you determine the `pensions` and `benefits-non-disability` assignments for this line?

File: .github/CODEOWNERS
Comment (rmtolmach) [inline .github/CODEOWNERS:574]: This will need to be updated again since [Benefits-Team-1](https://github.com/orgs/department-of-veterans-affairs/teams/benefits-team-1) is not active and only one person.

**Review by rmtolmach — COMMENTED** [review]
Thanks for cleaning this up!!

There are a few places where you're adding `Benefits-Team-1`. Those will need to be updated again once we can find more granular team assignments.

Also, let's get a review from lighthouse-banana-peels.

File: .github/CODEOWNERS
Comment (RachalCassity) [reply .github/CODEOWNERS:389]: Thanks!

File: .github/CODEOWNERS
Comment (RachalCassity) [reply .github/CODEOWNERS:402]: I looked up persistent_attachment in CODEOWNERS

File: .github/CODEOWNERS
Comment (RachalCassity) [reply .github/CODEOWNERS:574]: There has to be another owner. I will look into other claim documentation used in CODEOWNERS

File: .github/CODEOWNERS
Comment (RachalCassity) [reply .github/CODEOWNERS:937]: There are many lines where banana-peels is still present.

File: .github/CODEOWNERS
Comment (RachalCassity) [reply .github/CODEOWNERS:455]: Yes, the same owner for all

Comment (jerekshoe) [reply .github/CODEOWNERS:455]: These probably shouldn't be all the same owner. My team (benefits-management-tools-be) should probably own `payment_service.rb` but a lot of this is related to the benefits-non-disability team I think. Regardless, Benefits-Team-1 is not active and shouldn't be used

File: .github/CODEOWNERS
Comment (rmtolmach) [reply .github/CODEOWNERS:574]: The update doesn't necessarily have to happen now. We can get benefits folks to do it.

Comment (jerekshoe) [reply .github/CODEOWNERS:574]: This should be the benefits-non-disability team

File: .github/CODEOWNERS
Comment (RachalCassity) [reply .github/CODEOWNERS:455]: Thank you @jerekshoe

File: .github/CODEOWNERS
Comment (RachalCassity) [reply .github/CODEOWNERS:574]: Thank you!!

File: .github/CODEOWNERS
Comment (RachalCassity) [reply .github/CODEOWNERS]: Fixed!

**Review by rmtolmach — APPROVED** [review]

File: .github/CODEOWNERS
Comment (rmtolmach) [inline .github/CODEOWNERS:918]: Please add https://github.com/orgs/department-of-veterans-affairs/teams/backend-review-group to the entry
Offending line: modules/avs @department-of-veterans-affairs/va-cto-health-products

Comment (RachalCassity) [reply .github/CODEOWNERS]: DONE!

**Review by rmtolmach — APPROVED** [review]
🎉

**Pull request merged**
